skipper.go

```diff
@@ -1298,6 +1298,12 @@ func run(o Options, sig chan os.Signal, idleConnsCH chan struct{}) error {
 		Tracer:       tracer,
 	}
 
+	oo := auth.OidcOptions{
+		Timeout:      o.OAuthTokenintrospectionTimeout,
```

Comment:
This sounds a bit odd to re-use. I think we should add another option for this OpenID Connect config.

Reply:
SGTM

Reply:
done please review @szuecs